agent/agentssh/agentssh.go

```diff
@@ -60,6 +61,14 @@ const (
 	// MagicSessionTypeEnvironmentVariable is used to track the purpose behind an SSH connection.
 	// This is stripped from any commands being executed, and is counted towards connection stats.
 	MagicSessionTypeEnvironmentVariable = "CODER_SSH_SESSION_TYPE"
+	// ContainerEnvironmentVariable is used to specify the target container for an SSH connection.
+	// This is stripped from any commands being executed.
+	// Only available if CODER_AGENT_DEVCONTAINERS_ENABLE=true.
+	ContainerEnvironmentVariable = "CODER_CONTAINER"
```

Comment:
Should we be more specific about the container here? Is it only Devcontainer?

Reply:
I'm not forcing it to be :) It can be any container but the UI and `coder show` should only show devcontainers.

Reply:
ok, so in theory a smart Coder user can run any container this way?

Reply:
Yep, in theory. This is still 'early access' so I'm choosing to not restrict things like this until it makes sense to do so.